nextflow.config

```diff
@@ -1,8 +1,12 @@
 params {
     run_id = 1
+    help = false
     ref_ver = "hg19"
     bed_filter = ""
     exome_filter = false
+    
+    // Will add exceptions in validations.
+    ref_dir = "NOT_PROVIDED"
```

Comment:
it would be better to leave it empty.

Reply:
I'll make that change in the PR I'll be making for the parameter validation [issue-49](https://github.com/LiuzLab/AI_MARRVEL/issues/49).